src/SConscript

```diff
@@ -662,43 +670,229 @@ def BuildRelease(target, source, env):
+                {'wix': wix_path,
+                 'distdir': dist_dir,
+                 'sub': "promo"}
+            print "Using Command: " + command
+            subprocess.check_call(command)
+
+            command = '"%(wix)s\\candle.exe" -nologo -dWINLIBPATH=%(winlibpath)s -dPlatform=%(arch)s -d%(sub)sVar=%(distdir)s\%(sub)s -arch %(arch)s -out build\wix\subdirs\%(sub)s.wixobj build\wix\subdirs\%(sub)s.wxs' % \
+                {'wix': wix_path,
+                 'winlibpath': mixxx_winlib_path,
+                 'arch': winArch,
+                 'distdir': dist_dir,
+                 'sub': "promo"}
+            print "Using Command: " + command
+            subprocess.check_call(command)
+
+        # Handle QT's imageformats helper DLLs if dynamic QT
+        imageformats = "no"
+        if os.path.exists(os.path.join(dist_dir,"imageformats")) and not build.static_qt:
+            imageformats = "yes"
+            command = '"%(wix)s\\heat.exe" dir %(distdir)s\%(sub)s -nologo -sfrag -suid -ag -srd -cg %(sub)sComp -dr %(sub)sDir -out build\wix\subdirs\%(sub)s.wxs -sw5150 -var var.%(sub)sVar' % \
+                {'wix': wix_path,
+                 'distdir': dist_dir,
+                 'sub': "imageformats"}
+            print "Using Command: " + command
+            subprocess.check_call(command)
+
+            command = '"%(wix)s\\candle.exe" -nologo -dWINLIBPATH=%(winlibpath)s -dPlatform=%(arch)s -d%(sub)sVar=%(distdir)s\%(sub)s -arch %(arch)s -out build\wix\subdirs\%(sub)s.wixobj build\wix\subdirs\%(sub)s.wxs' % \
+                {'wix': wix_path,
                  'winlibpath': mixxx_winlib_path,
-                 '64bit': buildwin64,
-                 'staticdeps': staticdeps}
-            print "Using command: " + command
+                 'arch': winArch,
+                 'distdir': dist_dir,
+                 'sub': "imageformats"}
+            print "Using Command: " + command
+            subprocess.check_call(command)
+
+        # Harvest main DLL from install dir
+        command = '"%(wix)s\\heat.exe" dir %(distdir)s -nologo -sfrag -suid -ag -srd -cg mainDLLCompGroup -dr INSTALLDIR -out build\wix\subdirs\mainDLL.wxs -sw5150 -var var.SourceDir -t build\wix\only-dll.xslt' % \
+            {'wix': wix_path,
+             'distdir': dist_dir}
+        print "Using Command: " + command
+        subprocess.check_call(command)
+
+        command = '"%(wix)s\\candle.exe" -nologo -dWINLIBPATH=%(winlibpath)s -dPlatform=%(arch)s -dSourceDir=%(distdir)s -arch %(arch)s -out build\wix\subdirs\mainDLL.wixobj build\wix\subdirs\mainDLL.wxs' % \
+            {'wix': wix_path,
+             'winlibpath': mixxx_winlib_path,
+             'arch': winArch,
+             'distdir': dist_dir}
+        print "Using Command: " + command
+        subprocess.check_call(command)
+
+        # Harvest main PDB from install dir if they exist
+        isPdb = "no"
+        if bundle_pdbs and len(glob.glob(dist_dir + "\*.pdb")) > 0:
```

Comment:
just a minor nit -- I think it's more "Pythonic" to say: ```if list:``` instead of ```if len(list) > 0:```

e.g.
```if bundle_pdbs and glob.glob(os.path.join(dist_dir, "*.pdb")):```